README.md

```diff
@@ -1,3 +1,57 @@
 # KubeRobo
 
+KubeRobo will systematically demonstrate SIG Robotics’ end-to-end cloud-native robotics solution. It will serve as a comprehensive guide to all open source projects under the SIG Robotics umbrella, covering their scope, roadmap, and feature plans.
 
+![Introduce](./docs/images/intro.png)
+
```

Comment:
miss **contribution guidelines**, **motivation** and **use case**

Reply:
I think, for this repo kuberobo, there may not be contribution, guidelines and use case. Motivation can be added, I'm done.